File: src/main/java/com/example/FleetSystem/model/Vendor.java
package com.example.FleetSystem.model;
import lombok.*;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Builder
@Entity
public class Vendor {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String vendorName;
    private String officeLocation;
    private String attachments;
    private boolean status;

}

File: finance-ui/fms-finance/src/app/modal/vendor.ts
export interface Vendor {
    id:number|null|undefined,
    vendorName:string|null|undefined,
    officeLocation:string|null|undefined,
    contactPersonList: Array<
          {
          id:number|null|undefined,
          name:string|null|undefined,
          email:string|null|undefined,
          phoneNumber:string|null|undefined
          }>
       
    
    attachments:string|null|undefined,
}